Build.props

```diff
@@ -67,6 +67,11 @@
     <!--Overide the default full debug symbols regardless of configuration-->
     <DebugType>pdbonly</DebugType>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(MSBuildProjectExtension)' == '.vbproj'">

```

Comment:
why do we need to process "vb" again?

Reply:
There is 1-2 vb projects, can we remove all vb projects from e2e? Those are not in build earlier also, can we remove from solution/ any references

Reply:
Are these just test projects, testing the libraries from VB?

Reply:
Yes E2e test for vb, but those were/are not in ADO, should we remove those projects